supporting_scripts/analysis-of-endpoint-connections/src/main/java/de/tum/cit/endpointanalysis/EndpointAnalyzer.java

```diff
@@ -0,0 +1,125 @@
+package de.tum.cit.endpointanalysis;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class EndpointAnalyzer {
+
+    private static String EndpointAnalysisResultPath = "endpointAnalysisResult.json";
+
+    private static final Logger logger = LoggerFactory.getLogger(EndpointAnalyzer.class);
+
+    public static void main(String[] args) {
+        analyzeEndpoints();
+        printEndpointAnalysisResult();
+    }
+
+    /**
+     * Analyzes endpoints and matches them with REST calls.
```

Comment:
Maybe it is easier to understand like this? Without the context of the thesis, the description would be hard to understand for me because I would have used `endpoint` and `REST calls` rather interchangeable 😅 

```suggestion
     * Analyzes server side endpoints and matches them with client side REST calls.
```